src/containers/ConceptPage/EditConcept.tsx

```diff
@@ -44,6 +56,7 @@ const EditConcept = ({ conceptId, isNewlyCreated, licenses, selectedLanguage, t,
   return (
     <>
       <HelmetWithTracker title={`${concept.title} ${t('htmlTitles.titleTemplate')}`} />
+      {/* @ts-ignore */}
```

Comment:
Fant en mulig løsning for conceptform

```
export default compose<ComponentType<Props & tType>>(
  injectT,
  withRouter,
  connect(undefined, mapDispatchToProps),
)(ConceptForm);
```

Reply:
> Det ligger en del problemer rundt `CreateConcept`, da konseptet som passeres inn i `ConceptForm` ikke er fullverdig. Akkurat nå er alternativene å la `ConceptForm` ha en ts-ignore i `CreateConcept` eller å skrive om `ConceptForm` til å godta et partial concept. Sistnevnte vil muligens kreve noe arbeid. Er det noen andre alternativer man kan vurdere?

Det mest korrekte hadde nok vært å få ConceptForm til akseptere et ikke-komplett concept.